### DISCUSSION TITLE
QSelect multiple show only nth selected options and hide remaining

### DISCUSSION THREAD

**Discussion by zentoac** [quasarframework/quasar#11391] [Ideas / Proposals] [answered]
Hello,

when you have a multiple QSelect not too large, if you select many options, they are shown in the select container but they are arranged to start a new line... this is very unsightly.
It would be nice an option to show only nth selected options and hide remaining, just like the "size" option of bootstrap-select (https://developer.snapappointments.com/bootstrap-select/options/).

Thanks

**Comment by pdanpdan** [comment]
Use the slot, Luke
You can use selected or selected-item slots

**Reply by zentoac** [reply]
Oh yeah, cool!
May the force be with you! :)

**Reply by zentoac** [reply]
Well... actually is feasible but my custom message is repeated for every selected option. Any idea to show it just once?

![image](https://user-images.githubusercontent.com/70266119/142378951-58fbdd02-8d04-4e9f-a07c-a3f764927c6f.png)

This is the code:

```
          <q-select
            dense
            clearable
            multiple
            use-chips
            v-model.lazy="agent"
            input-debounce="150"
            use-input
            :options="agents"
            label="Agents"
            filled
            emit-value
            map-options
          >
            <template v-slot:selected-item="scope">
              {{ agent && agent.length === 1 ? scope.opt.label : agent.length + ' of ' + agents.length }}
            </template>
          </q-select>
        </div>
```

**Comment by pdanpdan** [comment]
selected-item is repeated for each item. selected is used only once

I'm on the phone right now, but if you put what you did in a codepen I'll make some adjustments to it

**Reply by zentoac** [reply]
Here it is... https://codepen.io/zentoac/pen/qBXgmGe

Thanks!!

**Comment by pdanpdan** [comment, marked as answer]
Different ways of showing the selected options:
https://codepen.io/pdanpdan/pen/abyXVBo?editors=1010

**Reply by zentoac** [reply]
Oh wow!!!! The good old live-saver "if" :)
Thank you very much!!!
